javacutil/src/main/java/org/checkerframework/javacutil/trees/TreeBuilder.java

```diff
@@ -103,10 +103,11 @@ public MemberSelectTree buildIteratorMethodAccess(ExpressionTree iterableExpr) {
       // Remove captured type variable from a wildcard.
       if (elementType instanceof Type.CapturedType) {
         elementType = ((Type.CapturedType) elementType).wildcard;
+        TypeElement iteratorElt = (TypeElement) modelTypes.asElement(iteratorType);
+        assert iteratorElt != null
+            : "@AssumeAssertion(nullness): element not found for " + iteratorType;
```

Comment:
After `@AssumeAssertion`, the string should be a justification for why the condition can never be true -- explain why iteratorElt is always null.  But, is the test backwards?